src/components/command-bar/commands/search/components/dialog-body/index.tsx

```diff
@@ -45,28 +64,77 @@ const SearchCommandBarDialogBodyContent = ({
 		return generateSuggestedPages(currentProductTag?.id as ProductSlug)
 	}, [currentProductTag])
 
-	return currentInputValue ? (
-		<div className={s.tabsWrapper}>
-			<Tabs
-				showAnchorLine={false}
-				initialActiveIndex={contentType === 'tutorials' ? 1 : 0}
-				variant="compact"
-			>
-				<Tab heading="Documentation" icon={<IconDocs16 />}>
+	/**
+	 * Generate an object used to render all of the Tab elements to preselect the
+	 * Tab for the CurrentContentType.
+	 */
+	const tabsBySearchableContentType = useMemo<
+		Record<SearchableContentType, SearchableContentTypeTab>
+	>(() => {
+		return {
+			docs: {
+				heading: 'Documentation',
+				icon: <IconDocs16 />,
+				content: (
 					<DocumentationTabContents
 						currentProductTag={currentProductTag}
 						suggestedPages={suggestedPages}
 					/>
-				</Tab>
-				<Tab heading="Tutorials" icon={<IconLearn16 />}>
+				),
+			},
+			tutorials: {
+				heading: 'Tutorials',
+				icon: <IconLearn16 />,
+				content: (
 					<TutorialsTabContents
 						currentProductTag={currentProductTag}
 						tutorialLibraryCta={generateTutorialLibraryCta(currentProductTag)}
 					/>
-				</Tab>
-			</Tabs>
-		</div>
-	) : (
+				),
+			},
+			integrations: {
+				heading: 'Integrations',
+				icon: <IconPipeline16 />,
+				content: (
+					<IntegrationsTabContents currentProductTag={currentProductTag} />
+				),
+			},
+		}
+	}, [currentProductTag, suggestedPages])
+	const searchableContentTypes = Object.keys(tabsBySearchableContentType)
+	const activeTabIndex =
+		contentType === 'global' ? 0 : searchableContentTypes.indexOf(contentType)
+
+	if (currentInputValue) {
+		return (
+			<div className={s.tabsWrapper}>
+				<Tabs
+					showAnchorLine={false}
+					initialActiveIndex={activeTabIndex}
+					variant="compact"
+				>
+					{searchableContentTypes.map((contentType: SearchableContentType) => {
+						if (
+							contentType === 'integrations' &&
+							!SHOULD_RENDER_INTEGRATIONS_TAB
```

Comment:
I think there's an issue here, where we need to filter out the integrations tab when the `currentProductTag` isn't one of the valid slugs. See the photo where consul results show the integrations tab, it would make more sense to me for this tab to only render when a product has integrations available. 

If we want to adjust, maybe we could handle the product filtering at the `tabsBySearchableContentType` object generation level and not as a part of the render function 💭 

<img width="940" alt="Screen Shot 2023-02-01 at 3 10 56 PM" src="https://user-images.githubusercontent.com/36613477/216187529-c6e6338b-ac6a-421b-970e-9911c791f644.png">

Reply:
I guess I can see the argument for having it for consistency's sake though, so if this was already discussed / decided I get it!

Reply:
💭 Very related - when you tab over to that Integrations tab under Consul, for example, there's a `See all Consul integrations` link which when followed results in an error (and at best i think would 404).

So +1 to maybe hiding the tab, or if the tab is to be shown consistently, maybe show "coming soon" for products within integrations activated or something? (Or at least just not rendering the `See all` link).

Reply:
This was just oversight! We can easily hide the tab if there are no integrations for the `currentProductTag`, and come back to the UX part later. 😊